v2/parser/parse.go

```diff
@@ -637,6 +637,10 @@ func (p *Parser) convertSignature(u types.Universe, t *gotypes.Signature) *types
 
 // walkType adds the type, and any necessary child types.
 func (p *Parser) walkType(u types.Universe, useName *types.Name, in gotypes.Type) *types.Type {
+	var out *types.Type
+	defer func() {
+		out.InitMultiverse(u)
```

Comment:
I just realized someone needs to call this on all of the builtin types.  This means that there can be only 1 Universe because the builtins are global variables, which have a single multiverse pointer each.  That makes me sad.

Can you think of any alternative that is not so limited?

Is the cure worse than the sickness?  I don't like that PointerTo(T) != PointerTo(T) but if there were two universes active T != T, anyway...

Have to think on this...